Blog/GamingGuruBlog.Web/GamingGuruBlog.Web/UIConverter.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using GamingGuruBlog.Domain;
+using GamingGuruBlog.Domain.Interfaces;
+using GamingGuruBlog.Web.Models;
+
+namespace GamingGuruBlog.Web
+{
+    public class UIConverter
+    {
+        private IServices _services;
+
+        public UIConverter(IServices _passedInServices)
+        {
+            _services = _passedInServices;
+        }
+
+
+        public void AddNewBlogPost(BlogPostVM newBlogPost)
+        {
+
+        }
+
```

Comment:
You're on the right track 👍 

This should have a single static method though....something that takes in a BlogPostVM and converts it to a BlogPost domain object.